### PR TITLE
Added .today() method

### PR DESCRIPTION
**Pull request by nxtpl**
### TLDR:
Added **.today()** method that resets calendar view to a current date;

### Full story:
When using this plugin on a website with multiple calendars, there was a problem - when really old date was picked, the rest calendars remembered it. So it was needed to change the year and month manually. Of course, there is a TODAY button, but users expected a fresh new calendar and submitted forms with wrong dates.
The plugin missed the functionality to reset calendar view only (without removing picked date from input field). To avoid confusion, I named the method **.today()**, as it don't alter the date field.
Also, added a test case,.

### CODE REVIEW

**Comment by nxtpl** [comment]
Hi, will it be integrated into the main version?

**Comment by DanielRuf** [comment]
Hi @nxtpl,

We are currently working on v5. You can already try an early v5 version. Can you check if it is missing there?

If so please create a new PR, otherwise we can close this one as resolved.

**Comment by DanielRuf** [comment]
Also please do not commit to your master branch and create a separate `feature/...` branch.
